pimd/pim_bsm.c

```diff
@@ -1146,17 +1145,13 @@ static bool pim_bsm_parse_install_g2rp(struct bsm_scope *scope, uint8_t *buf,
 		}
 		/* Extract Group tlv from BSM */
 		memcpy(&grpinfo, buf, sizeof(struct bsmmsg_grpinfo));
+		grp_addr = grpinfo.group.addr;
 
-		if (PIM_DEBUG_BSM) {
-			char grp_str[INET_ADDRSTRLEN];
-
-			pim_inet4_dump("<Group?>", grpinfo.group.addr, grp_str,
-				       sizeof(grp_str));
+		if (PIM_DEBUG_BSM)
 			zlog_debug(
-				"%s, Group %s  Rpcount:%d Fragment-Rp-count:%d",
-				__func__, grp_str, grpinfo.rp_count,
+				"%s, Group %pPAs  Rpcount:%d Fragment-Rp-count:%d",
+				__func__, &grp_addr, grpinfo.rp_count,
 				grpinfo.frag_rp_count);
-		}
 
 		buf += sizeof(struct bsmmsg_grpinfo);
 		offset += sizeof(struct bsmmsg_grpinfo);
```

Comment:
next 5 lines after this (not shown) need to be fixed (`AF_INET` + `IPV4_MAX_BITLEN`)

Reply:
Done